pages/common/csvsort.md

```diff
@@ -0,0 +1,20 @@
+# csvsort
+
+> Sorts CSV files.
+> Included in csvkit.
+
+- Sort a CSV file by column 9:
+
+`csvsort -c {{9}} {{data.csv}}`
+
+- Sort a CSV file by the "name" column in descending order:
+
+`csvsort -r -c {{name}} {{data.csv}}`
+
+- Sort a CSV file by columns 2, then by column 4:
+
+`csvsort -c {{2,4}} {{data.csv}}`
+
+- Sort a CSV file without inferring data types:
```

Comment:
It comes in handy from time to time, especially if you want to pass the CSV off to someone else (particularly a non-developer) or publish it or something.